app/api/government_service_data_api/calls_received_metric.rb

```diff
@@ -47,6 +47,26 @@ def not_provided?
     ].all? { |item| item == NOT_PROVIDED }
   end
 
+  def unspecified
+    return NOT_APPLICABLE if not_applicable? || not_provided?
```

Comment:
This reads oddly to me, as in "why is `NOT_APPLICABLE` being returned if the metric is `not_provided?`?".

However, I assume, it doesn't matter as if the whole metric is "not provided" then we don't render individual metric items. But then in that case, why is the guard statement needed?

Reply:
You're right, we probably don't need to guard against not_provided? here as it won't be shown.  Just me being over defensive.